src/chainparams.cpp

```diff
@@ -100,8 +100,8 @@ class CMainParams : public CChainParams {
 
         // Deployment of block-final miner commitment transaction.
         consensus.vDeployments[Consensus::DEPLOYMENT_BLOCKFINAL].bit = 1;
-        consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nStartTime = 1199145601; // January 1, 2008
-        consensus.vDeployments[Consensus::DEPLOYMENT_LOCKTIME].nTimeout = 1230767999; // December 31, 2008
+        consensus.vDeployments[Consensus::DEPLOYMENT_BLOCKFINAL].nStartTime = 1562068800; // July 2, 2019
+        consensus.vDeployments[Consensus::DEPLOYMENT_BLOCKFINAL].nTimeout = 1587038400; // April 16, 2020
```

Comment:
I see that you fix my previous comment in f761b39a52f4b257707244570b1ebd4125948595. Perhaps not even add the nStartTime/nTimeout values in the initial commit, as they are wrong anyway?